app/src/main/res/layout/table_recylcer_item.xml

```diff
@@ -16,7 +16,15 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:padding="5dp"
+        android:visibility="gone"
         android:id="@+id/data_item"
         />
-
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="5dp"
```

Comment:
Use dimen resource. Also, please change the padding to a standard value. Can 4dp be used? :)

Reply:
doing the changes